scale/mesos_api/tasks.py

```diff
@@ -95,6 +95,9 @@ def _create_docker_task(task):
     mesos_task = _create_base_task(task)
     mesos_task.container.type = mesos_pb2.ContainerInfo.DOCKER
     mesos_task.container.docker.image = task.docker_image
+    # add parameters
+    for k, v in task.docker_params:
+        mesos_task.container.docker.parameters.add(key=k, value=v)
```

Comment:
Does this work with Mesos 0.21? Just want to make it sure it wasn't a new interface or something.